src/subscription_manager/cli_command/abstract_syspurpose.py

```diff
@@ -173,8 +173,9 @@ def _is_provided_value_valid(self, value):
         # First check if the the value is in the valid_fields
         ret = False
         valid_fields = self._get_valid_fields()
+
```

Comment:
Minor nitpick: I'd avoid changes (like whitespace changes) unrelated to the actual bug fix.